querymonitor/querymonitor.pl

```diff
@@ -83,6 +84,7 @@ sub Main_loop_Iteration{
     }
 	my $ts = time();
     $opt{DEBUG} and print "DEBUG:$loop_count:$json_string\n";
+	$json_string !~/{/ and die "ERROR: $json_string";
```

Comment:
Would it be overkill to check more thoroughly that this is json?  like a jq-ish json_pp, module, sort of validation?

Reply:
Perl JSON is not a core module - I'm trying to avoid dependencies, so they don't have to install extra stuff.